### ISSUE TITLE
Web component: Card In Card - Card Eyebrow text "Label" is not identified for the screen reader user.

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#5504]
## Detailed description

On JAWS FF & VO iPhone safari

Issue:
Card Eyebrow text "Label" is not identified for the screen reader user.

## Additional information
**- This component was developed by Putra.  @RaphaelAmadeu The issue has been assigned to you to work with Putra on this fix.  This fix has been assigned to the March release and will be included in the fix pack scheduled to be created by Ignacio by Noon ET.**

![image.png](https://images.zenhubusercontent.com/5e21a7b470118a498429a20e/89d9c083-4c92-4363-8b68-66ddc3c18c1b)

**Comment by jeffchew** [comment]
Circling back on this, there was a discussion with @proeung regarding the current approach with the eyebrow treatment and its usage in other cards. This traces back to the following issue:

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/4062

From Putra:
```...my understanding is that the aria-hidden attribute that you guys have right now within the <dds-card-eyebrow> (see https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/card/card-eyebrow.ts#L30) full-fills the requirement of the ticket mentioned above. Perhaps, this is a matter of informing the decision that was made to whoever is running the screen reader tests (JAWS or VoiceOver) that this result is false positive?```

From this, we have decided to pull this out of the release, and will address either via documentation or perhaps targeting the element as `role="presentation"` instead.

FYI @RobertaJHahn @RaphaelAmadeu @ljcarot @andysherman2121 @shixiedesign

**Comment by shixiedesign** [comment]
On the call with Praveen and Ram this morning, Ram clarified that the need here is for "Tab" key to continue focus only the card title, but still allow "Up" and "Down" arrow keys to move focus to other texts to be announced. This means we need to shuffle the markup a bit and the update should be made on the `Card` component so all things dependent on it gets the update. This been an issue for awhile it is tracked here ---

**Comment by proeung** [comment]
@jeffchew Thanks for looping me in this issue thread. Personally, I think it's important to make the DOM order match the visual order of the component and have these text be readable by screen-readers, however, I do understand that this direction might result in the verbose reading issue that @shixiedesign mentioned in this thread - https://ibm-marketing-comms.slack.com/archives/G01JR9YHU3W/p1615906169004600?thread_ts=1615900683.000200&cid=G01JR9YHU3W.

Since this is an issue (`<dds-card-eyebrow>`) that touches a lot of card components and not just the `Card in Card`, I figured that the solution/fix makes more sense coming from your team. Regardless, let me know if there's anything that I can do to help out as well.

**Comment by RobertaJHahn** [comment]
Moved this fix to the April release, but added the Card issue (#4613 ) as a blocker.  An engineer should check to see whether this bug is fixed by the Card update mentioned in issue (#4613 )

**Comment by RobertaJHahn** [comment]
Ariella's comment posted to issue (#4613 ).  Moving this issue to sprint 21-08.  

_We just confirmed with the team the changes that will be made for this issue to be closed, so next sprint I will finish this issue. 5504 and mostly all the other card accessibility bugs require this fix being made._

**Comment by Praveen-sr** [comment]
@ariellalgilmore Ram re tested this on both JAWS and Voiceover on the [ link ]( https://ibmdotcom-webcomponents.s3.us-east.cloud-object-storage.appdomain.cloud/deploy-previews/6642/index.html?path=/story/overview-getting-started--page)  - Working as expected.
